include/View.hpp

```diff
@@ -49,7 +49,7 @@ class View : public ViewListeners
   FixedPoint<-4, int> x, y;
   std::unique_ptr<Popup> popup;
 
-  std::array<int16_t, 2u> previous_size;
+  std::array<int, 2u> previous_size;
```

Comment:
This will increase the size of the class. The difference shouldn't be significant but that's the reason why int16_t was used. Also this may cause some unecessary conversions within the window tree.

Reply:
This variable has one and only one use. When you're make a window "floating" we need the initial size of the window, before it gets into the Window Tree. I made it quickly a long time ago

Reply:
Ok, I don't think we need full ints but as I said this is minor unless we have conversions.